fineract-provider/src/main/java/org/apache/fineract/infrastructure/jobs/service/JobRegisterServiceImpl.java

```diff
@@ -118,6 +120,8 @@ public void executeJob(final ScheduledJobDetail scheduledJobDetail, String trigg
                 scheduler.addJob(jobDetail, true);
                 scheduler.triggerJob(jobKey, jobDataMap);
             }
+        } catch (JobIsNotFoundOrNotEnabledException e) {
+            throw e;
```

Comment:
This is unnecessary... unless you wanna log it, but then the log entry is missing

Reply:
It is not, because below that we turn every Exception into PlatformInternalServerException, but I didn't want to do this with this exception.
Log has been added though